spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -402,17 +374,6 @@ describe('The super header navigation', function () {
 
       expect(document.activeElement).toEqual($lastNavLink)
     })
-
-    it('when the last search menu link is focussed and the tab key is pressed the search menu is closed', function () {
-      $searchMenu.removeAttribute('hidden')
-      $lastSearchLink.focus()
-      window.GOVUK.triggerEvent($lastSearchLink, 'keydown', { keyCode: 9, cancelable: true })
-
-      expect($searchMenu.hasAttribute('hidden')).toEqual(true)
-      expect($searchMenuButton.getAttribute('aria-expanded')).toEqual('false')
-      expect($searchMenuButton.getAttribute('aria-label')).toEqual('Show search menu')
-      expect($searchMenuButton.classList).not.toContain('gem-c-layout-super-navigation-header__open-button')
-    })
```

Comment:
I think that instead of deleting the references to the removed links here we would need to update this test and the related component (layout-super-navigation-header) to reference all focusable elements, not just links - in this case `input` and `button` elements. The reason is that by removing the links and without doing this we lose some of the recently added behaviour: the search menu is closed when a keyboard-only user tabs through it and on to the next page element. 

My feeling is though that if this is too much extra work to get this particular change shipped then we could make a separate ticket to do that later as it's something that was added as part of the current round of a11y work but wasn't actually picked up in the audit so could be considered a nice to have - I don't think it would stop a keyboard-only user from navigating the page. 

If that is what we do then maybe we should also remove the relevant parts of the component itself, though again that might not be critical if we are going to return to that component to do some more work on it. 

Hope that makes sense but let me know if we should discuss further.

Reply:
We have subsequently decided to release this as it is and create a new ticket to refactor the previous addition to the `layout-super-navigation-header` component to restore the functionality lost here and make it more sturdy. 

- The previous addition to the `layout-super-navigation-header` component was made in [Adjust keyboard functionality of dropdown menu [L]](https://trello.com/c/ke3jvHve/2400-adjust-keyboard-functionality-of-dropdown-menu-l)
- The new ticket to refactor and improve the work done in that ticket is in [Refactor and improve recent change to layout-super-navigation-header component](https://trello.com/c/zhgxoNEK/2485-refactor-and-improve-recent-change-to-layout-super-navigation-header-component)